octobot_trading/traders/trader.py

```diff
@@ -51,13 +51,13 @@ def __init__(self, config, exchange_manager):
 
         if not hasattr(self, 'simulate'):
             self.simulate = False
-
         self.is_enabled = self.__class__.enabled(self.config)
-        self.logger.debug(f"{'Enabled' if self.is_enabled else 'Disabled'} on {self.exchange_manager.exchange_name}")
 
     async def initialize_impl(self):
+        self.is_enabled = self.is_enabled and self.exchange_manager.is_trading
```

Comment:
Finally we disable the trader if the exchange manager is not trading ?

Reply:
I think it makes more sense to have the trader being deactivated when its exchange is not trading rather than having an activated trader on a no-trading exchange

Reply:
Yes it seems logic, I was just thinking about unexpected effects of disabling Trader. I think it should be ok :+1:

Reply:
For now I see no issue :)